tools/misc/tpm2_checkquote.c

```diff
@@ -267,7 +267,7 @@ static bool parse_selection_data_from_file(FILE *pcr_input,
     }
 
     if (le64toh(pcrs->count) > ARRAY_LEN(pcrs->pcr_values)) {
-        LOG_ERR("Malformed PCR file, pcr count cannot be greater than %zu, got: %zu",
+        LOG_ERR("Malformed PCR file, pcr count cannot be greater than %zu, got: %" PRIu64 " ",
                 ARRAY_LEN(pcrs->pcr_values), le64toh(pcrs->count));
```

Comment:
For fixed width 64 bit values, PRIu64 is the correct specifier. %zu should be reserved for size_t.

@idesai why is their a le64toh call? We have all that endian swapping code to avoid using endian.h. I avoided endian.h due to portability issues, but that might be moot using clang to compile on Windows. If we're going to use endian.h, I have no problems with it that I forsee, other than doing:
```C
#if defined(__FreeBSD__)
#include <sys/endian.h>
#else
#include <endian.h>
#endif
```
and then we can delete a lot of the endian swapping code and just use endian.h and perhaps consider something like:
  - https://gist.github.com/PkmX/63dd23f28ba885be53a5

for dealing with windows....

Reply:
Sounds good. I like this direction. I will create a ticket.

Reply:
Created #2241 to track these changes.